sdk/dnsresolver/Microsoft.Azure.Management.DnsResolver/src/Microsoft.Azure.Management.DnsResolver.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.DnsResolver</PackageId>
+		<Description>Provides management capabilities for Azure DNS Resolver.</Description>
+		<Version>1.0.0</Version>
```

Comment:
Since this looks like a new package, you should start with a 'prerelease' version number: 1.0.0-beta.1

Reply:
Updated.